python_webapp2_vanilla/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM jin09/webapp2
+
+MAINTAINER Gautam Jain <gautam.jain9@yahoo.com>
+
+RUN mkdir /home/src/
+
+COPY / /home/src/
+
+RUN rm -f /home/src/Dockerfile
+
+RUN pip install -r /home/src/requirements.txt
+
+CMD ["python", "/home/src/main.py"]
```

Comment:
Also, in the directory name of vanilla webapp2 use hyphens instead of underscore (as every other folder is hyphenated). So `python-webapp2-vanilla` instead of `python_webapp2_vanilla`.